server/models/meteorological.py

```diff
@@ -1,18 +1,20 @@
-from sqlalchemy import Column, Float, Integer, String
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
 
 from server.db.base_class import Base
 
 
-class Historical_Meteorological(Base):
-    __tablename__ = "Historical Meteorological Data"
+class Meteorological(Base):
+    __tablename__ = "Meteorological"
```

Comment:
Set this lowercase for consistency with all our other tables.

`__tablename__ = "meteorological"`